entrypoint.sh

```diff
@@ -88,7 +88,7 @@ if [ -n "$GITHUB_TOKEN" ]; then
   else
     message=$(/message $name 0)
   fi
-  /notify-pr.sh "$message" $GITHUB_TOKEN
+  /notify-pr.sh "$message" $GITHUB_TOKEN $name
```

Comment:
nit: could we call it also `preview_name` as in the script?

Reply:
Yeah, it would make more sense to me as well, but didn't want to modify the existing code more than needed, so I decided to leave `name` as-is [as defined above](https://github.com/okteto/deploy-preview/blob/d06f169dfa3cfb497a210808097bf4d7d4baba38/entrypoint.sh#L4)